test/mock_transport_test.exs

```diff
@@ -0,0 +1,93 @@
+defmodule Janus.MockTransportTest do
+  use ExUnit.Case
+
+  alias Janus.MockTransport
+
+  @request_response_pairs [
+    {
+      %{
+        janus: :create
+      },
+      %{
+        "janus" => "success",
+        "data" => %{"id" => "session id"}
+      }
+    },
+    {
+      %{
+        janus: :keepalive
+      },
+      %{
+        "janus" => "ack"
+      }
+    },
+    {
+      %{
+        janus: :keepalive
+      },
+      %{
+        "janus" => "ack"
+      }
+    }
+  ]
+
+  describe "MockTransport should" do
```

Comment:
What if request which is not expected is sent?

Reply:
As for now module will throw and exception if no corresponding response is found.